optional_plugins/html/avocado_result_html/__init__.py

```diff
@@ -119,11 +119,13 @@ def tests(self):
         results_dir = self.results_dir(False)
         for tst in self.result.tests:
             formatted = {}
-            formatted['name'] = tst['name']
+            formatted['uid'] = tst['name'].uid
```

Comment:
One difference I spotted here is: one the test ID shown on the UI (and on the previous HTML report) we have a "0 padded" number, such as `01`.  With this change, we now have `1`.

Not sure if you caught that, and I'm also not sure if that is a problem.